en/render-pipeline/builtin-pipeline.md

```diff
@@ -0,0 +1,25 @@
+# Builtin Rendering Pipeline
+
+As of version 3.1, the builtin rendering pipeline includes builtin-forward and builtin-deferred. The rendering pipeline can be switched via **Project->Project Settings->Project Data->RenderPipeline**, and restart the editor after the switch to take effect.
+
+<img src="./image/setting.png" width=560 height=384></img>
```

Comment:
why not use Markdown?

Reply:
> why not use Markdown?

I want to specify the size of the image, but Markdown does not support it well.